backend/src/main/java/dccs/academy/entities/CertificateEntity.java

```diff
@@ -0,0 +1,114 @@
+package dccs.academy.entities;
+
+import jakarta.persistence.*;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+
+@Entity
+@Table(name = "certificates",schema = "certificates")
+public class CertificateEntity {
+    public enum CertificateType {
```

Comment:
Enum will be used through app, not just here so please move it to separate file. Think about package to store enums in.